compiler/damlc/tests/daml-test-files/Generics.daml

```diff
@@ -1,6 +1,9 @@
 -- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
+-- @INFO Redundant case
```

Comment:
Where is the redundant case here? I’m having trouble seeing any case statement.

Reply:
As we have determined, this is in generated code. Fixing upstream (testing now). Will withdraw this PR most likely. Stand by.